cudaaligner/tests/Test_AlignmentImpl.cpp

```diff
@@ -161,8 +161,10 @@ TEST_P(TestAlignmentImpl, AlignmentFormatting)
 {
     FormattedAlignment formatted_alignment = alignment_->format_alignment();
     std::string query                      = formatted_alignment.first;
+    std::string pairing                    = formatted_alignment.pairing;
     std::string target                     = formatted_alignment.second;
```

Comment:
remove these local std::strings and use `formatted_alignment` directly. That will save 3 lines of code. :)

Reply:
Thanks, I will an operator== to the struct!

Reply:
no, an `==` is not necessary. You usually don't want to compare these objects.